jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -1280,7 +1315,8 @@ void java_object_factoryt::allocate_nondet_length_array(
     assignments,
     "nondet_array_length",
     from_integer(0, java_int_type()),
-    max_length_expr);
+    max_length_expr,
+    true);
```

Comment:
:no_entry_sign: Pull into a const variable so documenting what this floating boolean is

Reply:
Got rid of the boolean.